safebrowser.go

```diff
@@ -405,30 +409,34 @@ func (sb *SafeBrowser) LookupURLs(urls []string) (threats [][]URLThreat, err err
 	// TODO: There are some optimizations to be made here:
```

Comment:
I think the whole TODO could be deleted now. Along with the one you are fixing here, the DB now does proper retrying according to the specified backoff schedule. So there's not really a situation in which we should trigger an update while doing a lookup. On receiving a failure indicating a DB issue, the caller could call WaitUntilReady(ctx) and get the right behavior (Ideally we will have public errors that make it easy to tell if it is a DB issue versus other types of errors).